pennylane/ops/functions/generator.py

```diff
@@ -29,15 +29,15 @@ def _generator_observable(gen, op):
     as provided by the original gate.
     """
     if isinstance(gen, (qml.Hermitian, qml.SparseHamiltonian)):
-        if not op.inverse:
+        if not getattr(op, "inverse", False):
```

Comment:
how come this was changed?

Reply:
`Operator` itself doesn't have `inverse`, so `Adjoint` does not have an `inverse`.

I could potentially just add a dummy property for now.

Reply:
And reverted since I'm now mixing in the `AdjointOperation` class.